modules/programs/msmtp-accounts.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs }:
```

Comment:
shouldn't we mov ethe file to the accounts folder instead ?

Reply:
For now I prefer to keep the program specific account submodule near the program module main file. This since I imagine one would typically work on a program module and want easy access to its associated files.